rule-manager/conf/logback.xml

```diff
@@ -0,0 +1,41 @@
+<!-- https://www.playframework.com/documentation/latest/SettingsLogger -->
+<configuration>
```

Comment:
I wonder if there's a way for us to share logback configs (esp. WRT kinesis) – we can explore in another PR!

Reply:
Definitely a good idea. As you say, I wouldn't want to block the progress of this prerequisite work, so one for another PR.